packages/dm-core-plugins/src/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { TPlugin } from '@development-framework/dm-core'
+import Editor from './explorer/Editor'
+import { PluginComponent } from './yaml-view/YamlPlugin'
+import { TabsContainer } from './tabs/TabsContainer'
+import { MermaidComponent } from './mermaid-plugin/MermaidComponent'
+import { JobInputEdit } from './job/JobInputEdit'
+import { JobControlWrapper } from './job/JobControlWrapper'
+import { FormComponent } from './form/FormComonent'
```

Comment:
```suggestion
import { FormComponent } from './form/FormComponent'
```
This is wrong in the filename too

Reply:
good catch! Will fix